README.md

```diff
@@ -38,7 +38,7 @@ Chain data in `./chains` folder is generated using steps described here: https:/
 > yarn build
 
 > cd packages/protocol
-# NOTE: Following command is found in: sdk/contractkit/package.json, at some point the desired --upto flag can change
+# NOTE: Following command is found in: sdk/contractkit/package.json, at some point the desired `--upto` flag can change
```

Comment:
this can be removed since the command this refers to us being removed from contractkit `test:reset`